doc/source/intro.rst

```diff
@@ -31,7 +31,7 @@ Start a python interpreter and import the required PyACP packages:
         get_example_file,
     )  # This is only required for the tutorial
 
-To not pollute the filesystem, we are going to create a temporary directory where we can download the files:
+To not pollute the filesystem, create a temporary directory for the example files:
```

Comment:
@SRaimondi I had to sneak in an edit to the `intro.rst` here, since the "Documentation style" check was failing.

The root cause is that "warning" level errors (here: avoiding the use of "we") are inconsistently reported. I think this may be fixed by setting `fail-level: warning` on the `doc-style` action, but that option is not available in a released version of ansys/actions yet.